.github/workflows/main.yml

```diff
@@ -47,7 +50,19 @@ jobs:
     - uses: actions/checkout@v4
     - name: Install Rust
       run: bash ci/install-rust.sh ${{ matrix.rust }} ${{ matrix.target }}
+    - name: Install npm
+      if: matrix.os != 'windows-latest'
+      uses: actions/setup-node@v3
+      with:
+        node-version: 20
+    - name: Install browser-ui-test
+      if: matrix.os != 'windows-latest'
+      run: npm install browser-ui-test@"${BROWSER_UI_TEST_VERSION}"
+    - name: Build and run tests (+ GUI)
+      if: matrix.os != 'windows-latest'
+      run: cargo test --locked --target ${{ matrix.target }} --test gui
     - name: Build and run tests
+      if: matrix.os == 'windows-latest'
       run: cargo test --locked --target ${{ matrix.target }}
```

Comment:
This seems to disable the default tests on non-windows runners. `cargo test --locked --target ${{ matrix.target }} --test gui` *only* runs the GUI tests.

Reply:
Ah indeed! Good catch.